Makefile

```diff
@@ -0,0 +1,24 @@
+
+deps-dev:
+	pip-compile requirements/base.in requirements/dev.in -o requirements/dev.txt
+
+install-dev: pip-tools
+	pip-sync requirements/dev.txt
+
+deployment-deps:
+	pip-compile requirements/base.in requirements/deployment.in -o requirements/deployment.txt
+
+install:
```

Comment:
Shouldn't this depend on `pip-tools`?

Reply:
i think it should. :) good one, will fix soon.